cypress/support/index.js

```diff
@@ -13,8 +13,14 @@
 // https://on.cypress.io/configuration
 // ***********************************************************
 
-// Import commands.js using ES2015 syntax:
+import '@cypress/code-coverage/support';
+
 import './commands';
 
-// Alternatively you can use CommonJS syntax:
-// require('./commands')
+Cypress.on('uncaught:exception', (err, runnable, promise) => {
+  // returning false here prevents Cypress from failing the test
+  // this is necessary to accept wanted error from mirage to fail the test
+  if (promise) {
+    return false;
+  }
+});
```

Comment:
Could this potentially block us from finding real errors?

Reply:
Potentially yes. I will add some console so the user can more easily debug.